doc/classes/Skeleton3D.xml

```diff
@@ -213,6 +213,12 @@
 				This is helper function to make using [method Transform3D.looking_at] easier with bone poses.
 			</description>
 		</method>
+		<method name="init_bone_poses">
```

Comment:
Would it not make more sense to call it `reset_bone_poses()` ?

Reply:
Renamed. Also Skeleton3DEditor's function names has been changed.